### PR TITLE
Clarify that non-objects aren't allowed as VC graphs.

### PR DESCRIPTION
**Pull request by msporny**
This PR attempts to address issue #1365 by clarifying that non-object values are not allowed for Verifiable Credential Graphs.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1390.html" title="Last updated on Dec 21, 2023, 6:13 PM UTC (3356cbd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1390/edcbf7e...3356cbd.html" title="Last updated on Dec 21, 2023, 6:13 PM UTC (3356cbd)">Diff</a>

### CODE REVIEW

File: index.html
Comment (dlongley) [inline index.html:2292]: Having the sentence "These types of objects" follow the avoidance of doubt sentence was a little confusing, so we could avoid this with a parenthetical:

```suggestion
objects (for the avoidance of doubt, the values MUST NOT be non-object values such as
numbers, strings, or URLs). These types of objects are called
```

File: index.html
Comment (dlongley) [inline index.html:2294]: Hmm, an enveloped verifiable credential is not itself secured, rather its contents are. With the current language, it would need to be enveloped again (and then this problem infinitely recurses), whereas a VC with an embedded proof is secured. To try and cover both of these cases, I propose this slight tweak here:

```suggestion
<a href="#verifiable-credential-graphs">verifiable credential graphs</a> and
MUST express information that is secured using a
<a href="#securing-mechanisms">securing mechanism</a>.
```

**Review by dlongley — APPROVED** [review]
Approving assuming my suggestions will be applied. Thanks!

**Review by iherman — APPROVED** [review]

**Review by andresuribe87 — APPROVED** [review]

**Review by decentralgabe — APPROVED** [review]

File: index.html
Comment (TallTed) [inline index.html:2291]:
```suggestion
objects (to be clear, the values MUST NOT be non-object values such as
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Review by brentzundel — APPROVED** [review]
approved with Ted's suggestion

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-12-19-vcwg-special#section2-7) on 2023-12-19

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 2.7. Clarify that non-objects aren't allowed as VC graphs. (pr vc-data-model#1390)

_See github pull request [vc-data-model#1390](https://github.com/w3c/vc-data-model/pull/1390)._




**Brent Zundel:** "Clarify that non-objects aren't allowed as VC graphs". This has a broad set of approvers, one request for changes.  
… From Ted who is bringing editorial clarity with his suggestion. I expect this to be merged without trouble by the end of the week.  
… Happy for comments if there are any.  
… No one on the queue here either.  

</details>

**Review by longpd — APPROVED** [review]

**Comment by msporny** [comment]
Normative, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
